tests/integration/actions/run/test_stdout_tmux.py

```diff
@@ -116,6 +99,23 @@ class ShellCommand(UiTestStep):
         ).join(),
         present=["from.ansible.cfg", "ok=1"],
     ),
+    ShellCommand(
+        comment="run playbook with enable-prompts and show prompting",
+        user_input=StdoutCommand(
+            cmdline=f"{playbook_path} -k --enable-prompts",
+            execution_environment=True,
+        ).join(),
+        present=["password:"],
```

Comment:
we should be able to set

`search_within_response = "SSH password"`

here, since it is a property of the ShellCommand class